server/member/member.go

```diff
@@ -297,6 +298,14 @@ func (m *Member) getMemberLeaderPriorityPath(id uint64) string {
 	return path.Join(m.rootPath, fmt.Sprintf("member/%d/leader_priority", id))
 }
 
+func (m *Member) GetDCLocationPathPrefix() string {
+	return path.Join(m.rootPath, dcLocationConfigEtcdPrefix)
+}
+
+func (m *Member) GetDCLocationPath(id uint64) string {
+	return path.Join(m.rootPath, dcLocationConfigEtcdPrefix, fmt.Sprint(id))
```

Comment:
```suggestion
	return path.Join(m.GetDCLocationPathPrefix(), fmt.Sprint(id))
```

I think this is better.